.changeset/purple-poets-smoke.md

```diff
@@ -0,0 +1,13 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Fixed usage of the `@tracked` modifier in several components:
```

Comment:
```suggestion
Fixed instances where arguments are passed into tracked properties at declaration:
```

Reply:
or some other way, to highlight that not the `@tracked` modifier is the issue here, but the pattern of declaring initial values for properties using arguments